templates/forms/default/data.html.twig

```diff
@@ -14,8 +14,10 @@
                         {% block field_value %}
                             {% if field.type == 'checkboxes' %}
                                 <ul>
-                                    {% for value in form.value(scope ~ field.name) %}
-                                        <li>{{ field.options[value]|e }}</li>
+                                    {% set use_keys = field.use is defined and field.use == 'keys' %}
+                                    {% for key,value in form.value(field.name) %}
```

Comment:
looks like scope is missing

Reply:
Yes it is, well noticed! Think it was related to copying from #121. I remember noticing it now. Obviously forgot all about it very soon after! Fix coming.